File: jscript/k00.parameters.js
// create subroutine
function celsius(fahrenheit) {
  temperature = (fahrenheit - 32) * 5 / 9; // calculate to internatioanl temp
  temperature = temperature.toFixed(1);    // lower degree of significance

  // output results
  WScript.echo("The Celsius temperature is " + temperature + " degrees.");
}

temperature = 73;     // set temp in fahrenheit
celsius(temperature); // call subroutine

File: lessons/compiled_lang/go/src/k00.parameters.go
package main

import "fmt"

func celsius(fahrenheit float64) {
	temperature := (fahrenheit - 32.0) * 5 / 9
	fmt.Printf("The Celsius temperature is %.1f degrees.\n", temperature)
}

func main() {
	temperature := 73.0
	celsius(temperature)
}
